src/websockets/binary_websockets.es6

```diff
@@ -149,14 +152,30 @@ const needs_authentication = (data) => {
    return false;
 };
 
+const website_status = (data) => {
```

Comment:
@4p00rv I don't think belongs to this file. Either we write a separate JS/class file where this notification is handled(we can have it inside navigation folder or we move this code to navigation.es6
binary_websockets.es6 is to purely deal with websockets and dispatching events.